frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -295,3 +295,51 @@ Feature: Pipeline Runs
               And user clicks on Start
              Then user will be redirected to Pipeline Run Details page
               And user will see PVC Workspace "workspace-ex" mentioned in the VolumeClaimTemplate Resources section of Pipeline Run Details page
+
+
+        @regression, @manual
+        Scenario: Pipeline Run details page with passed pipeline run having finally task
+            Given pipeline with finally task having a passed pipeline run is present
+              And user is on Pipeline Run details page of passed pipeline run
+             When user scrolls over each of the finally task
+             Then user can see finally tasks arranged in parallel with each task as passed
```

Comment:
These two lines doesn't seem right to me, even if the user fails to scroll, they will still be able to see the finally tasks. Maybe interchange these lines and replace when with And
```suggestion
             And user can see finally tasks arranged in parallel with each task as passed
             When user hovers over each of the finally tasks
```